source/install/install-rhel-8.rst

```diff
@@ -3,169 +3,99 @@
 Install Mattermost on RHEL
```

Comment:
Just a note that the code blocks in-line with text do not seem to have the style applied (i.e. the colour)

Reply:
On https://docs.mattermost.com/install/install-tar.html you can see the in-line code is the same blue as the code boxes.

Reply:
Thank you!